1.10.2/src/main/java/net/creeperhost/creeperhost/gui/mpreplacement/CreeperHostEntry.java

```diff
@@ -69,6 +79,12 @@ public void setSelected(int p_178011_1_, int p_178011_2_, int p_178011_3_)
      */
     public boolean mousePressed(int slotIndex, int mouseX, int mouseY, int mouseEvent, int relativeX, int relativeY)
     {
+        if(isInRect(exitX, exitY, 60, 10, mouseX, mouseY )){
+            Config.getInstance().setMpMenuEnabled(false);
+            CreeperHost.instance.saveConfig();
+            this.mc.displayGuiScreen(new GuiMultiplayer(null));

```

Comment:
This should return the current `parentScreen` from the current `GuiMultiplayer`, other wise I think it will crash and/or burn when you to to the server select menu and then try to go back to the MainMenu